app/javascript/src/components/Dashboard/Notes/Card.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Tag, Avatar, Tooltip, Dropdown } from "neetoui";
+import { useTranslation } from "react-i18next";
+
+import { formatCreationDate } from "./utils";
+
+const Card = ({ title, description, created_at: createdAt }) => {
+  const { fullDate, relativeDate } = formatCreationDate(createdAt);
+  const { t } = useTranslation();
```

Comment:
Keep the hooks at the very top unless the value from the other statements are required to invoke the hook.